coderd/provisionerjobs_test.go

```diff
@@ -2,16 +2,116 @@ package coderd_test
 
 import (
 	"context"
+	"encoding/json"
 	"testing"
+	"time"
 
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 
 	"github.com/coder/coder/v2/coderd/coderdtest"
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/coderd/database/dbgen"
+	"github.com/coder/coder/v2/coderd/database/dbtestutil"
+	"github.com/coder/coder/v2/codersdk"
 	"github.com/coder/coder/v2/provisioner/echo"
 	"github.com/coder/coder/v2/provisionersdk/proto"
 	"github.com/coder/coder/v2/testutil"
 )
 
+func TestProvisionerJobs(t *testing.T) {
+	t.Parallel()
+
+	// encode := func(v interface{}) []byte {
+	// 	b, err := json.Marshal(v)
+	// 	require.NoError(t, err)
+	// 	return b
+	// }
+
+	// db, ps := dbtestutil.NewDB(t,
+	// 	dbtestutil.WithDumpOnFailure(),
+	// 	//nolint:gocritic // Use UTC for consistent timestamp length in golden files.
+	// 	dbtestutil.WithTimezone("UTC"),
+	// )
+	// client, _, coderdAPI := coderdtest.NewWithAPI(t, &coderdtest.Options{
+	// 	IncludeProvisionerDaemon: true,
+	// 	Database:                 db,
+	// 	Pubsub:                   ps,
+	// })
+	// owner := coderdtest.CreateFirstUser(t, client)
+	// _, memberUser := coderdtest.CreateAnotherUser(t, client, owner.OrganizationID)
+
+	db, ps := dbtestutil.NewDB(t, dbtestutil.WithDumpOnFailure())
```

Comment:
Disgusting idea: use `dbtestutil.NewDBWithSQLDB` and just insert the raw rows you want 😅